### PR TITLE
#170947557 User Roles Settings

### PR DESCRIPTION
**Pull request by Baraka-Mugisha**
#### What does this PR do?
- Adds option of updating user roles

#### Description of Task to be completed?
- add roles field in the user database
- add internationalized messages
- seed a default super admin
- add the update roles controller and routes

#### How should this be manually tested?
- clone the repo
- run `npm install` to install all dependencies
- log in as a seeded super admin
- run the server and go to `api/v1/setRoles/` route in POSTMAN and change an existing user's roles

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- [#170947557](https://www.pivotaltracker.com/story/show/170947557)

#### Screenshots (if appropriate)
-N/A

### CODE REVIEW

File: src/services/localesServices/locales/en.json [hunk 1]
```diff
@@ -5,5 +5,23 @@
...
+	"The user doesn't exist": "The user doesn't exist",
+	"The email is required": "The email is required",
+	"Invalid value": "Invalid value",
+	"manager": "manager",
+	"travel team member": "travel team member",
+	"requester": "requester",
+	"travel administrator": "travel administrator",
+	"super administrator": "super administrator",
+	"the acceptable roles are manager, travel team member, requester, travel administrator, super administrator": "the acceptable roles are manager, travel team member, requester, travel administrator, super administrator",
+	"The user is arleady a %s": "The user is arleady a %s",
```
Comment (Mnickii) [inline src/services/localesServices/locales/en.json]: already instead of arleady

Comment (Baraka-Mugisha) [reply src/services/localesServices/locales/en.json]: it is changed now

**Review by Mnickii — APPROVED** [review]

**Review by bbaime98 — APPROVED** [review]

**Comment by bbaime98** [comment]
if you can add a screenshot it would make clear what functionality this feature adds.

**Review by Ntare22 — COMMENTED** [review]
Add some screenshots please

**Review by Ntare22 — APPROVED** [review]

**Comment by Ntare22** [comment]
Please update your PR description.

**Pull request merged**